### ISSUE TITLE
Agenda Doesn't Work in Sublime Text 4.

### ISSUE THREAD

**Issue by r4dian** [ihdavids/orgextended#85]
**Describe the bug**

In Sublime Text 3, I have the Org Extended agenda working correctly.

In sublime text 4, the agenda view only displays the first 10 lines with the calendar. 

This occurs in fresh installations of the editor(s). There are no other packages installed except those installed as Org Extended dependencies, and the Org Extended settings are the same between both versions.

**To Reproduce**
Steps to reproduce the behaviour:
1. Instal Sublime Text 3 and Sublime Text 4 on the same machine.
2. Open each version, and install Package Control in both.
3. Use package control to install Org Extended & restart both versions of Sublime Text.
4. Go to `Preferences > Package Settings > Org Extended > Settings` and set `"orgDirs":[]` to the correct array of directories.
5. Restart both again.
6. Press `alt-o, o, a` to load the agenda in each.

**Expected behaviour**
Should behave the same in both versions.

**Screenshots**

![Screenshot 2024-02-28 122050](https://github.com/ihdavids/orgextended/assets/1264174/74f7b7e4-efc1-41ec-a4d9-93d4d23402ba "Left (light theme): Sublime Text 4 / Right (dark theme): Sublime Text 4")

**Platform:**
Windows 10 x64 and Windows 11 Pro

**Sublime debug console output**
```
DPI mode: per-monitor v2
startup, version: 4169 windows x64 channel: stable
executable: /C/Program Files/Sublime Text/sublime_text.exe
application: /C/Program Files/Sublime Text
working dir: /C/Program Files/Sublime Text
packages path: /C/Users/████.█████/AppData/Roaming/Sublime Text/Packages
state path: /C/Users/████.█████/AppData/Roaming/Sublime Text/Local
zip path: /C/Program Files/Sublime Text/Packages
zip path: /C/Users/████.█████/AppData/Roaming/Sublime Text/Installed Packages
ignored_packages: ["Vintage"]
Unable to parse color value var(bgcol) at Packages/OrgExtended/OrgExtended-Light.sublime-color-scheme:415:18
pre session restore time: 0.155579
startup time: 0.208579
first paint time: 0.213579
reloading plugin Default.arithmetic
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.colors
reloading plugin Default.comment
reloading plugin Default.convert_color_scheme
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading python 3.3 plugin OrgExtended.asettings
reloading plugin Default.history_list
reloading plugin Default.html_print
reloading plugin Default.indentation
reloading python 3.3 plugin OrgExtended.beancount
reloading plugin Default.install_package_control
reloading python 3.3 plugin OrgExtended.orgagenda
reloading plugin Default.keymap
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.rename
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Package Control.plugin
reloading python 3.3 plugin OrgExtended.orgbuiltinresources
reloading python 3.3 plugin OrgExtended.orgcapture
reloading python 3.3 plugin OrgExtended.orgcheckbox
reloading python 3.3 plugin OrgExtended.orgclocking
reloading python 3.3 plugin OrgExtended.orgdatepicker
reloading python 3.3 plugin OrgExtended.orgdaypage
reloading python 3.3 plugin OrgExtended.orgdb
reloading python 3.3 plugin OrgExtended.orgduration
reloading python 3.3 plugin OrgExtended.orgdwim
SETTINGS IS NULL? IS THIS BEING CALLED BEFORE PLUGIN START?
reloading python 3.3 plugin OrgExtended.orgdynamicblock
reloading python 3.3 plugin OrgExtended.orgediting
reloading python 3.3 plugin OrgExtended.orgexporter
reloading python 3.3 plugin OrgExtended.orgextended
reloading python 3.3 plugin OrgExtended.orgextension
reloading python 3.3 plugin OrgExtended.orgfolding
reloading python 3.3 plugin OrgExtended.orghtml
reloading python 3.3 plugin OrgExtended.orghtmlexporter
reloading python 3.3 plugin OrgExtended.orginsertselected
reloading python 3.3 plugin OrgExtended.orginternalediting
reloading python 3.3 plugin OrgExtended.orginternalhelpers
reloading python 3.3 plugin OrgExtended.orglatex
reloading python 3.3 plugin OrgExtended.orglinks
reloading python 3.3 plugin OrgExtended.orglist
reloading python 3.3 plugin OrgExtended.orgmouse
reloading python 3.3 plugin OrgExtended.orgnavigation
reloading python 3.3 plugin OrgExtended.orgneovi
reloading python 3.3 plugin OrgExtended.orgnotifications
reloading python 3.3 plugin OrgExtended.orgnumberedlist
reloading python 3.3 plugin OrgExtended.orgpandoc
reloading python 3.3 plugin OrgExtended.orgplist
reloading python 3.3 plugin OrgExtended.orgproperties
reloading python 3.3 plugin OrgExtended.orgreadtheorg
reloading python 3.3 plugin OrgExtended.orgrevealjs
reloading python 3.3 plugin OrgExtended.orgsourceblock
reloading python 3.3 plugin OrgExtended.orgswiper
reloading python 3.3 plugin OrgExtended.orgtableformula
reloading python 3.3 plugin OrgExtended.orgtableplot
reloading python 3.3 plugin OrgExtended.orgtimechart
reloading python 3.3 plugin OrgExtended.orgtrello
Install the Trello Package to add trello support to OrgExtended for SublimeText 
  >> No module named 'Trello'
reloading python 3.3 plugin OrgExtended.orguniqueview
reloading python 3.3 plugin OrgExtended.orgunittests
reloading python 3.3 plugin OrgExtended.orgxmlthemeparser
reloading python 3.3 plugin OrgExtended.packagecon
reloading python 3.3 plugin OrgExtended.pymitter
reloading python 3.3 plugin OrgExtended.simple_eval
reloading python 3.3 plugin Processing.Processing
reloading python 3.3 plugin Table Editor.table_base
reloading python 3.3 plugin Table Editor.table_border_syntax
reloading python 3.3 plugin Table Editor.table_emacs_org_mode_syntax
reloading python 3.3 plugin Table Editor.table_lib
reloading python 3.3 plugin Table Editor.table_lib_test
reloading python 3.3 plugin Table Editor.table_line_parser
reloading python 3.3 plugin Table Editor.table_multi_markdown_syntax
reloading python 3.3 plugin Table Editor.table_pandoc_syntax
reloading python 3.3 plugin Table Editor.table_plugin
reloading python 3.3 plugin Table Editor.table_plugin_test
reloading python 3.3 plugin Table Editor.table_re_structured_text_syntax
reloading python 3.3 plugin Table Editor.table_simple_syntax
reloading python 3.3 plugin Table Editor.table_textile_syntax
reloading python 3.3 plugin Table Editor.widechar_support
plugins loaded
Unable to parse color value var(bgcol) at Packages/OrgExtended/OrgExtended-Light.sublime-color-scheme:415:18
Package Control: Skipping automatic upgrade, last run at 2024-02-28 12:46:32, next run at 2024-02-28 20:46:32 or after
```
username blanked out with █

**Additional context**
~~Add any other context about the problem here.~~

**Comment by schultzter** [comment]
I just want to add that changing dates Next/Prev refreshes the display and everything shows up. Going back to the current date Prev/Next also works. It's just the initial opening of the Agenda that stops after the monthly calendars.

**Comment by ihdavids** [comment]
This is a really strange one. It seems the compound Agenda view inserts the text for each view one after another. When the panel FIRST opens it will only accept a single insert.

For now I have worked around it by setting up a compound command that will run, open the panel and then run the same command in draw mode and draw all the text. This seems to get around the problem but I am darned if I know what the issue is.

Should be fixed in 1.2.54
